libcxx/docs/DesignDocs/VisibilityMacros.rst

```diff
@@ -64,25 +64,13 @@ Visibility Macros
   ABI, we should create a new _LIBCPP_HIDE_FROM_ABI_AFTER_XXX macro, and we can
```

Comment:
Not attached to this line: you should update the `Overview` at the beginning of this file to reflect the latest state of things.